tests/ui/imports/issue-45829/rename-use-vs-extern.stderr

```diff
@@ -9,9 +9,8 @@ LL | use std as issue_45829_b;
    = note: `issue_45829_b` must be defined only once in the type namespace of this module
 help: you can use `as` to change the binding name of the import
    |
-LL - use std as issue_45829_b;
-LL + use std as other_issue_45829_b;
-   |
+LL | use std as other_issue_45829_b as issue_45829_b;
+   |         ++++++++++++++++++++++
```

Comment:
This doesn't seems right, before it was replacing the `as issue_45829_b` with `as other_issue_45829_b`; but now it's adding it *before* the `as issue_45829_b` making this suggestion invalid as we cannot have two `as` rename for a import.

Reply:
Fixed